app/assets/stylesheets/custom.css.scss

```diff
@@ -44,6 +44,45 @@ body {
   }
 }
 
+.smfont {
+  font-size: 0.9em;
+}
+.warning {
+  color: red;
+  padding-left: 30px;
+}
+
+.card {
+  background-color: #FFF;

```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`